assets/css/base/_sidebar.scss

```
@@ -42,47 +38,58 @@ $sidebar-width: 250px;
       padding-top: 1rem;
       padding-bottom: 1rem;
 
-      &:hover
-      {
+      &:hover {
         text-decoration: none;
         background: rgba(0, 0, 0, 0.2);
       }
 
-      &:active, &:focus
-      {
+      &:active, &:focus {
         text-decoration: none;
       }
 
       .user-notification-badge {
         display: none;
       }
-      .user-notification-badge-dropdown
-      {
+
+      .user-notification-badge-dropdown {
         display: none;
       }
     }
   }
 
   @include media-breakpoint-up(md) {
-    &:not(.inactive)
-    {
+    &:not(.inactive) {
       left: 0;
     }
   }
 }
 
 @include media-breakpoint-up(md) {
-  body.new-nav .container {
-    padding-left: 15px + $sidebar-width;
-  }
+  body {
+    &.new-nav {
+      .catro-tabs {
+        margin-left: $sidebar-width;
+      }
+
+      .container {
+        padding-left: 15px + $sidebar-width;
+
+        .catro-tabs {
+          margin-left: 0;
+        }
+      }
 
-  body.new-nav.modal-open .modal {
-    border-left: $sidebar-width solid transparent;
+
+
+      .modal-open .modal {
```

`&` missing. `body.new-nav.modal-open .modal` != `body.new-nav .modal-open .modal`
```suggestion
      &.modal-open .modal {
```